action.php

```diff
@@ -503,7 +503,7 @@ protected function generatePDF($cachefile, $event) {
         }
 
         // insert the back page
-        $body_end = $template['back'];
+        $body_end = $template['back'] ?? '';
```

Comment:
In general it is nicer if variables are always defined, in stead of check if these exists.
In this case this variable is forgotten in the `load_template()` function. Could you add it please? Please check if there are more variables returned that are possible not defined.

Reply:
Thanks for pointing this out. I haven't found any other undefined variables and added just `back` to the list in `load_template()`.

I amended the commit.